CHANGELOG.md

```diff
@@ -1,9 +1,28 @@
-# dbt_tiktok_ads version.version
+# dbt_tiktok_ads v0.8.0
+
+[PR #24](https://github.com/fivetran/dbt_tiktok_ads/pull/24) includes the following updates:
+## Schema Changes
+**4 total changes • 0 possible breaking changes
+| Table/Column                                      | Change Type | Old Name | New Name                                  | Notes                                                             |
+|---------------------------------------------------|-------------|----------|-------------------------------------------|-------------------------------------------------------------------|
+| tiktok_ads__campaign_country_report       | New Model   |          |  | New table that represents the daily performance of a campaign at the country/geographic region level.               |
+| stg_tiktok_ads__campaign_country_report_tmp       | New Model   |          |  | Temp model added for `campaign_country_report`.               |
+| stg_tiktok_ads__campaign_country_report           | New Model   |          |    | Staging model added for `campaign_country_report`.         |
+| objective_type, status, budget, budget_mode, created_at, is_new_structure           | New Columns   |          |    | New columns added to the stg_tiktok_ads__campaign_history model.         |
+
+## Feature Updates
+- Added the `tiktok_ads__campaign_country_report` end model and upstream staging models. See above for schema change details and new models added.
+  - For dbt Core users: If you do not sync this table or would like disable these new models you can disable the models by setting the  `tiktok_ads__using_campaign_country_report` variable to `false` in your `dbt_project.yml` file (`true` by default).
+- Included the `tiktok_ads__campaign_country_report_passthrough_metrics` passthrough variable in the above mentioned new staging models. Refer to the [README](https://github.com/fivetran/dbt_tiktok_ads/tree/main?tab=readme-ov-file#passing-through-additional-metrics) for more details.
+- Introduced the above mentioned new columns to the `stg_tiktok_ads__campaign_history` model.
```

Comment:
Is this repetitive with line 11 of the Schema Changes table?

Reply:
Agreed - added